src/replica/bulk_load/replica_bulk_loader.cpp

```diff
@@ -925,10 +966,11 @@ void replica_bulk_loader::report_bulk_load_states_to_primary(
     auto local_status = _status;
     switch (remote_status) {
     case bulk_load_status::BLS_DOWNLOADING:
-    case bulk_load_status::BLS_DOWNLOADED:
+    case bulk_load_status::BLS_DOWNLOADED: {
+        zauto_read_lock l(_lock);
```

Comment:
`process` and `status` read seem no need lock

Reply:
Those will be accessed by tasks executed in different threadpool, so they should be locked.